pkg/tsdb/azuremonitor/azuremonitor-datasource_test.go

```diff
@@ -500,7 +500,7 @@ func loadTestFile(name string) (AzureMonitorResponse, error) {
 	var azData AzureMonitorResponse
 
 	path := filepath.Join("testdata", name)
-	jsonBody, err := ioutil.ReadFile(path)
+	jsonBody, err := ioutil.ReadFile(filepath.Clean(path))
```

Comment:
After reviewing the gosec [issue thread](https://github.com/securego/gosec/issues/439), I think we should silence this warning, since `filepath.Clean` doesn't resolve any security issue and any security concerns around the folder path should be resolved when originally constructing it.